src/main/java/bio/terra/service/job/StairwayExceptionFieldsFactory.java

```diff
@@ -0,0 +1,103 @@
+package bio.terra.service.job;
+
+import bio.terra.common.exception.ErrorReportException;
+import bio.terra.service.job.exception.ExceptionSerializerException;
+import bio.terra.service.job.exception.JobExecutionException;
+import bio.terra.service.job.exception.StepExecutionException;
+import bio.terra.stairway.Step;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import org.apache.commons.lang3.StringUtils;
+
+public class StairwayExceptionFieldsFactory {
```

Comment:
If this is really a factory for `StairwayExceptionFields` then it should be written as static methods on that class. E.g.
```
  public static StairwayExceptionFields fieldsFromException(Exception ex) {
```
should be
```
  public static StairwayExceptionFields fromException(Exception ex) {
```
but declared on `StairwayExceptionFields` itself.